cdr_data_dictionary/generate_yaml.py

```diff
@@ -137,35 +144,24 @@ def sequentially_process_tab_contents(values, init_fields=None):
     return fields, concepts
 
 
-def process_tab_contents(values, id_index):
```

Comment:
this function was used by the `_write_yaml_dict` function only.  It was never used as part of the workflow.  so it was essentially "dead" code.  it was unreachable.  i felt like leaving it in would be confusing for new maintainers.  it had been left in with the thought that it might be used later.  but it isn't, so the risk seemed to outweigh the benefits.